assets/dev/js/frontend/frontend.js

```diff
@@ -76,7 +76,7 @@ class Frontend extends elementorModules.ViewModule {
 
 	getGeneralSettings( settingName ) {
 		if ( this.isEditMode() ) {
-			elementorCommon.helpers.softDeprecated( 'getGeneralSettings', '3.0.0', 'getKitSettings and remove the `elementor_` prefix' );
+			parent.elementorCommon.helpers.softDeprecated( 'getGeneralSettings', '3.0.0', 'getKitSettings and remove the `elementor_` prefix' );
```

Comment:
In which case `elementorCommon` is not available in edit mode?

Reply:
An author on preview/frontend.